packages/mobile/capacitor/plugins/secure-filesystem-access/android/src/main/java/org/iota/plugins/securefilesystemaccess/SecureFilesystemAccessPlugin.java

```diff
@@ -322,6 +322,45 @@ public void listProfileFolders(PluginCall call) {
         call.resolve(response);
     }
 
+    @PluginMethod
+    public void copyFile(PluginCall call) {
+        if (!call.getData().has("source")
+                || !call.getData().has("destination")) {
+            call.reject("source and destination is required");
+            return;
+        }
+        try {
+            String source = Objects.requireNonNull(call.getString("source"));
+            String destination = Objects.requireNonNull(call.getString("destination"));
+            File sourceFile = new File(source);
+            File destinationFile = new File(getContext().getFilesDir(), destination);
+            Files.copy(sourceFile.toPath(), destinationFile.toPath());
+            call.resolve();
+        } catch (Exception e) {
+            call.reject(Objects.requireNonNull(e.getCause()).toString());
+        }
+    }
 
+    @PluginMethod
+    public void deleteFile(PluginCall call) {
+        if (!call.getData().has("source")) {
+            call.reject("source is required");
+            return;
+        }
+        try {
+            String source = Objects.requireNonNull(call.getString("source"));
+            File fileName = new File(getContext().getFilesDir(), source);
+            if (fileName.exists()) {
+                boolean isDeleted = fileName.delete();
+                if (!isDeleted) {
+                    call.reject("Can't delete Stronghold file");
```

Comment:
I think that this error could have a generic message, otherwise detect the ".stronghold" file extension before throwing it
```suggestion
                    call.reject("Can't delete file");
```